packages/cfd/src/Components/props.types.ts

```diff
@@ -158,9 +158,9 @@ export type TTradingPlatformAccounts = {
     /**
      * Market type
      */
-    market_type?: 'financial' | 'synthetic';
+    market_type?: 'financial' | 'synthetic' | 'all';
```

Comment:
@hamza-deriv Same here.

Reply:
Can Update it but it will cause extra 'gaming'.
So have to leave this in my opinion